doc/version-history.rst

```diff
@@ -4,6 +4,11 @@
 Version History
 ===============
 
+v18.0.0
```

Comment:
The next scheduled release is v17.1.0 below. I think you should just add your change to the existing items, no need to create a new release.

Reply:
Fixed. Thanks.